### PR TITLE
Add versioning

### PR DESCRIPTION
**Pull request by rafahop**
### What

Add `/v1` prefix
Update swagger spec
Bump dependency

### How to review

Check changes make sense

### Who can review

Anyone

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ schemes:
 tags:
   - name: "public"
 paths:
-  /legacy:
+  /articles/legacy:
```
Comment (ghost) [inline swagger.yaml:15]: Quick question: the path should not include the 'v1'? i.e. /v1/articles/legacy

**Review by ghost — COMMENTED** [review]
Just the 1 question

**Review by ghost — COMMENTED** [review]
Just the 1 question

File: swagger.yaml (hunk 1 above)
Comment (rafahop) [reply swagger.yaml:15]: Yes, it's a bit further up, in line 9 (unchanged)

[basePath: "/v1"](https://github.com/ONSdigital/dp-articles-api/blob/cb913da034b8f43ad7a9354be5cea0e6f486c04c/swagger.yaml#L9)

**Review by ghost — APPROVED** [review]

**Pull request merged**